src/metabase/notification/task/send.clj

```diff
@@ -44,7 +44,6 @@
    (triggers/with-identity (send-notification-trigger-key subscription-id))
    (triggers/using-job-data {"subscription-id" subscription-id})
    (triggers/for-job send-notification-job-key)
-   (triggers/start-now)
```

Comment:
this was a mistake, but luckily it had no effect since with-schedule will override it.